x-pack/plugins/fleet/common/types/models/agent.ts

```diff
@@ -152,7 +152,18 @@ export interface ActionStatus {
   nbAgentsFailed: number;
   version?: string;
   startTime?: string;
-  type?: string;
+  type:
+    | 'POLICY_REASSIGN'
```

Comment:
Q: Are those the same as [AgentActionType](https://github.com/elastic/kibana/blob/0ca0717a292eabaa5ce790080f5836c83c5e0439/x-pack/plugins/fleet/common/types/models/agent.ts#L39-L47)? In that case you could use that type, or extend it if needed.

Reply:
Thanks, I missed there is an existing type. I'll update.